src/ui/App.tsx

```diff
@@ -131,18 +131,20 @@ function App(): JSX.Element {
   // of the stack.
   cards.reverse();
 
+  const noCardsStyleOrEmpty = StyleSheet.create({
```

Comment:
Is there a common pattern for splitting out constant styles from non-constant styles?

I don't love this, but I think I like it better than what was there before.

Reply:
What don't you like about this?

Reply:
I would have just put this on "style" btw. Maybe that's what you're asking about?

  const style = StyleSheet.create({
    view: {flex: 1, backgroundColor: '#eee'},
    noCardsOrEmpty:  cards.length ? {} : {justifyContent: 'center', alignItems: 'center'},	
  });

Or maybe:
  const noCardsOrEmptyStyle =  cards.length ? {} : {justifyContent: 'center', alignItems: 'center'},	
  const style = StyleSheet.create({
    view: {...{flex: 1, backgroundColor: '#eee'}, ...noCardsOrEmptyStyle},
  });